webapp/src/pages/ChatroomDetailsPage/ChatroomDetailsPage.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  ButtonSize,
+  ButtonVariant,
+  Header,
+  IconVariant,
+  MediumAvatar,
+  RowItem,
+  RowsListTemplate,
+  TextColor,
+  TextVariant,
+  TextWeight,
+  Text,
+} from '../../shared/components';
+import { AVATAR_EP_URL, CHAT_URL, CHATROOM_URL } from '../../shared/urls';
+import { useParams } from 'react-router-dom';
+import { useNavigation } from '../../shared/hooks/UseNavigation';
+import React, { useCallback, useState } from 'react';
+import './ChatroomDetailsPage.css';
+import { chatApi } from '../../shared/services/ApiService';
+import { useData } from '../../shared/hooks/UseData';
+import { ResponseError } from '../../shared/generated';
+import { Chatroom } from '../../shared/generated/models/Chatroom';
+import { useAuth } from '../../shared/hooks/UseAuth';
+import { ChatroomMemberWithUser } from '../../shared/generated/models/ChatroomMemberWithUser';
+import Loading from '../../shared/components/Loading/Loading';
+import { ENTRIES_LIMIT } from '../../shared/constants';
+import {
+  Query,
+  SearchContextProvider,
+} from '../../shared/context/SearchContext';
+import { ChatControllerGetChatroomMembersRequest } from '../../shared/generated/apis/ChatApi';
+import { useNotificationContext } from '../../shared/context/NotificationContext';
+
+export default function ChatroomDetailsPage() {
+  const { warn } = useNotificationContext();
+  const { chatroomId } = useParams();
+  const { goBack, navigate } = useNavigation();
+  const getChatroom = useCallback(
+    () => chatApi.chatControllerGetChatroomById({ id: chatroomId! }),
+    [chatroomId],
+  );
+  const { data: chatroom } = useData<Chatroom>(getChatroom);
+  const { authUser } = useAuth();
+  const isOwner: boolean = authUser?.id === chatroom?.ownerId;
+  const [ownerHasBeenWarned, setOwnerHasBeenWarned] = useState(false);
+  const mapChatMemberToRow = (member: ChatroomMemberWithUser): RowItem => {
+    return {
+      iconVariant: IconVariant.EDIT,
+      avatarProps: {
+        url: `${AVATAR_EP_URL}/${member.avatarId}`,
+        status: 'offline',
+        XCoordinate: member.avatarX,
+        YCoordinate: member.avatarY,
+      },
+      url: `${CHATROOM_URL}/${chatroomId}/member/${member.username}/edit`,
```

Comment:
thought: Currently, we use the same url and icon for all the members.
suggestion: What do you think about adding a TODO to deal with the case where the user is not an owner or administrator?

Reply:
done